File: PRACTICA PRIMER PARCIAL/Ejercicio10/Python/SecuenciaRepetida.py
def generar_secuencia(n):
    secuencia = []
    
    for i in range(1, n + 1):
        numero = i * 2
        secuencia.extend([numero, numero])
    
    return secuencia

n = int(input("Ingrese la cantidad de términos: "))

secuencia = generar_secuencia(n)
print("La secuencia generada es:", secuencia)

File: PRACTICA PRIMER PARCIAL/Ejercicio10/Java/SecuenciaRepetida.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class SecuenciaRepetida {
    public static List<Integer> generarSecuencia(int n) {
        List<Integer> secuencia = new ArrayList<>();

        for (int i = 1; i <= n; i++) {
            int numero = i * 2;
            secuencia.add(numero);
            secuencia.add(numero);
        }

        return secuencia;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Ingrese la cantidad de términos: ");
        int n = scanner.nextInt();

        List<Integer> secuencia = generarSecuencia(n);
        System.out.println("La secuencia generada es: " + secuencia);
    }
}